content/tutorials/drift-detection-and-remediation/index.md

```diff
@@ -0,0 +1,224 @@
+---
+title_tag: "Detect and Remediate Infrastructure Drift"
+title: "Detect and Remediate Infrastructure Drift"
+layout: single
+description: |
+   Learn to detect and remediate infrastructure drift with Pulumi Deployments.
+meta_desc: In this tutorial, learn to detect and remediate infrastructure drift in AWS using Pulumi Deployments and GitHub integration.
+meta_image: meta.png
+weight: 71
+summary: |
+   In this tutorial, you will learn how to detect and remediate infrastructure drift in AWS using Pulumi Deployments. You'll create an EC2 instance, introduce drift by making manual changes via the AWS CLI or console, and then use Pulumi Deployments to detect and correct the drift.
+you'll_learn:
+ - How to set up and configure Pulumi Deployments for drift detection
+ - How to introduce and identify infrastructure drift in an AWS environment
+ - How to remediate drift ensuring your cloud infrastructure matches your desired state
+prereqs:
+   - Completion of the [Getting Started](/docs/get-started/) guide or familiarity with the basics of the Pulumi workflow
+   - The [Pulumi CLI](/docs/install/)
+   - A [Pulumi Cloud account](https://app.pulumi.com/signup) and organization subscribed to the Enterprise or Business Critical editions
+   - An [Amazon Web Services](https://aws.amazon.com/) account, access key ID, and secret access key
+   - The [AWS CLI](https://aws.amazon.com/cli/)
+   - Git installed and a [GitHub account](https://github.com/) with admin rights to a repository or organization
+estimated_time: 15
+---
+
+[Drift Detection and Remediation](/docs/pulumi-cloud/deployments/drift/) is a feature of Pulumi Cloud that automates the detection and correction of drift in your cloud environments. It integrates seamlessly with Pulumi Deployments and can be incorporated into CI/CD workflows and tools using the [Pulumi CLI](/docs/cli/) or [Automation API](docs/using-pulumi/automation-api/). When changes occur outside of Pulumi, such as manual updates in the cloud provider's console, Pulumi reports these discrepancies. Auto-remediation can also be configured to refresh your last known desired state, correcting any drift without manual intervention.
```

Comment:
I think you should mention that you can run drift detection inside your own CICD system as well. It doesn't need to only be configured with Deployments. You should link to that section of the docs if people want to do that.

Reply:
thx, feedback incorporated.